src/index.html

```diff
@@ -1,13 +1,20 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Antisnake</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Antisnake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Antisnake</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="antisnake">
+      <div class="antisnake__block antisnake__block--block-1">1</div>
+      <div class="antisnake__block antisnake__block--block-2">2</div>
+      <div class="antisnake__block antisnake__block--block-3">3</div>
+      <div class="antisnake__block antisnake__block--block-4">4</div>
+      <div class="antisnake__block antisnake__block--block-5">5</div>
+      <div class="antisnake__block antisnake__block--block-6">6</div>
```

Comment:
```suggestion
      <div class="antisnake__block"></div>
      <div class="antisnake__block"></div>
      <div class="antisnake__block"></div>
      <div class="antisnake__block"></div>
      <div class="antisnake__block"></div>
      <div class="antisnake__block"></div>
```
якщо далі для зміни кольорів зробиш цикл: в цьому не буде необхідності

Reply:
loop has been done

Reply:
Right. Misunderstood it